db/write_controller.h

```diff
@@ -28,7 +28,7 @@ class WriteController {
         total_delayed_(0),
         total_compaction_pressure_(0),
         bytes_left_(0),
```

Comment:
It might just be me, but bytes_left is a bad name.  It is NOT the number of bytes_left to send but the BUDGET of bytes that can still be sent at the given rate.  At the minimum it merits a clarifying comment but a name change would also be helpful (byte_budget_left_?)